### ISSUE TITLE
Rule proposal: no async in describe callback

### ISSUE THREAD

**Issue by michielmulders** [lo1tuma/eslint-plugin-mocha#189]
According to this issue with [Mocha](https://github.com/mochajs/mocha/issues/2975) (Describe block with async function behaving weirdly #2975),
it is not recommended to use the async keyword in the describe callback argument.

Example piece of code to avoid:
```js
describe('yyy', async () => {

});
```

Preferred code:
```js
describe('yyy', () => {

});
```

Would you like to consider this rule?
Thanks for your time!

**Comment by lo1tuma** [comment]
Yes, such a rule makes definitely sense. There is already a PR which implements this, see #188.

**Comment by lo1tuma** [comment]
Fixed by #188 and released in v5.3.0.
